src/gpu/intel/ocl/simple_binary.cpp

```diff
@@ -26,6 +26,7 @@ namespace ocl {
 status_t simple_binary_t::pd_t::init_conf(impl::engine_t *engine) {
     const memory_desc_wrapper src0_d(src_md(0));
     const memory_desc_wrapper src1_d(src_md(1));
+    const memory_desc_wrapper src2_d(is_ternary_op() ? src_md(2) : dst_md());
```

Comment:
The check against `is_tenary_op()` should be unnecessary. If we are not in a ternary case, `src_md(2)` should return nullptr, and the memory descriptor wrapper will just use `glob_zero_md`. https://github.com/uxlfoundation/oneDNN/blob/6531ea5fc257bf375242ed0ef1729f684153ead4/src/common/memory_desc_wrapper.hpp#L44